readme.md

```diff
@@ -1,7 +1,7 @@
 # HTML form
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_html-form/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_html-form/report/html_report/)
+- [DEMO LINK](https://oleksandr-rudiuk.github.io/layout_html-form/)
+- [TEST REPORT LINK](https://oleksandr-rudiuk.github.io/layout_html-form/report/html_report/)
```

Comment:
paste these links to the description
https://www.loom.com/share/da672f3d083c4247960d02d43f7d527a

Reply:
and make them work. did you deploy the page? if you have questions, ask in the chat